src/with-children.js

```diff
@@ -4,14 +4,18 @@ export const withChildren = (
   Base: Class<HTMLElement> = HTMLElement
 ): Class<HTMLElement> =>
   class extends Base {
+    childrenChangedCallback: Function | void;
```

Comment:
doesn't flow support something more robust [for mixin definition like typescript](https://github.com/Hotell/reactive-typesafe-webcomponents/blob/vanilla-mixins/02-vanilla-lit-html/src/mixins/withRender.ts#L10) has? hey just asking, no flame war ✌️

Reply:
How would this help `childrenChangedCallback`? All you'd need to do is declare an interface and apply it to the `Class<>` annotation. I don't think that's entirely necessary, though, is it?